db/app.js

```diff
@@ -1,17 +1,25 @@
 const express = require("express");
 const app = express();
 
-const { viewCategories, getMessage } = require("./controllers/app.controller");
+const {
+  viewCategories,
+  getMessage,
+  viewReviews,
+} = require("./controllers/app.controller");
 
 app.use(express.json());
 
 app.get("/api", getMessage);
 
+app.get("/not-a-route");
```

Comment:
This doesn't seem to do anything here. There's no callback function as a parameter.